src/ArToolkitProfile.ts

```diff
@@ -112,7 +168,7 @@ export class ArToolkitProfile {
     /**
      * check if the profile is valid. Throw an exception is not valid
      */
-    checkIfValid() {
+    checkIfValid(): this {
```

Comment:
Not sure if this  method make sense, i will leave here for reference with old code.